pysite/views/api/bot/user.py

```diff
@@ -26,13 +26,18 @@
     }
 ])
 
+BANNABLE_STATES = ["preparing", "running"]
```

Comment:
do we need to modify this? if not, why not make it a tuple?

Reply:
Done.